packages/astro/src/core/dev/restart.ts

```diff
@@ -30,6 +30,7 @@ async function createRestartedContainer(
 }
 
 const configRE = new RegExp(`.*astro\.config\.((mjs)|(cjs)|(js)|(ts))$`);
+// TODO: use typegenDir
```

Comment:
I need help for this one, I guess we should generate the regex using a string and getting a relative path to root?